src/Logic/SubtitleFormats/SubRip.cs

```diff
@@ -275,5 +275,19 @@ public override List<string> AlternateExtensions
                 return new List<string> { ".wsrt" };
             }
         }
+
+        public override void RemoveNativeFormatting(Subtitle subtitle, SubtitleFormat newFormat)
+        {
+            // {i}Italic{/i}, {b}Bold{/b}, {u}Underline{/u}
+            var regex = new Regex(@"\{/?[ibu]\}");
+            var count = subtitle.Paragraphs.Count;
+            for (int i = 0; i < count; i++)
+            {
+                var p = subtitle.Paragraphs[i];
+                var text = regex.Replace(p.Text, string.Empty);
+                if (text != p.Text)
+                    p.Text = text;
+            }
+        }
     }

```

Comment:
Or, perhaps,
```
foreach (var p in subtitle.Paragraphs)
{
    p.Text = regex.Replace(p.Text, string.Empty);
}
```
[MSDN](https://msdn.microsoft.com/en-us/library/xwewhkd1%28v=vs.110%29.aspx): *If the regular expression pattern is not matched in the current instance, the method returns the current instance unchanged.*